### PR TITLE
Mock feature branch

### PR DESCRIPTION
**Pull request by cschroedl-gov**
Just testing out a third-party web-based code review tool.
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/cschroed-usgs%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23issuecomment-69756865%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-01-13T15%3A02%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20b2f6b5338bcb5e4f8cf0ece25255ace408406311%20README.md%208%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866485%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20doesn%27t%20this%20tool%20add%20the%20capability%20to%20ignore%20whitespace%20at%20the%20click%20of%20a%20button%3F%22%2C%20%22created_at%22%3A%20%222015-01-13T14%3A58%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-01-13T15%3A01%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-9%22%7D%2C%20%22Pull%20b2f6b5338bcb5e4f8cf0ece25255ace408406311%20src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866529%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20method%20call%20is%20unsafe%3B%20asking%20for%20an%20NPE%22%2C%20%22created_at%22%3A%20%222015-01-13T14%3A59%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-01-13T15%3A01%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3419837%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cschroed-usgs%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java%3AL33-48%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23issuecomment-69756865%22%2C%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866485%22%2C%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866529%22%2C%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866728%22%2C%20%22https%3A//github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1%23discussion_r22866744%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/cschroed-usgs'></a>
- [x] <a href='#crh-comment-Pull b2f6b5338bcb5e4f8cf0ece25255ace408406311 README.md 8'></a> &nbsp;<b><a href='https://github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1#discussion_r22866485'>File: README.md:L1-9</a></b>
- <a href='https://github.com/cschroed-usgs'></a> Why doesn't this tool add the capability to ignore whitespace at the click of a button?
- [x] <a href='#crh-comment-Pull b2f6b5338bcb5e4f8cf0ece25255ace408406311 src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java 10'></a> &nbsp;<b><a href='https://github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1#discussion_r22866529'>File: src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java:L33-48</a></b>
- <a href='https://github.com/cschroed-usgs'></a> this method call is unsafe; asking for an NPE

<a href='https://www.codereviewhub.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1?approve=0'></a>&nbsp;<a href='https://github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by cschroed-usgs** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cschroed-usgs/consul-registrator-maven-plugin/pull/1'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-This maven plugin permits registration of a service with a consul agent.
+This maven plugin permits registration of a service with a consul agent. 
 
 Requirements: a consul agent must be running and listening on the service host.
 
 Invocation:
-
+
```
Comment (cschroedl-gov) [inline README.md]: Why doesn't this tool add the capability to ignore whitespace at the click of a button?

File: src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java [hunk 2]
```diff
@@ -33,8 +33,15 @@ public static String buildScript(String customScript, String[] contextPaths){
     }
     
     public ConsulRegistratorMojoTest() {
+	//adding some comments
+	String anecdote = "blah";
+	int answerToEverything = newMethod(anecdote);
+    }
+    public static int newMethod(String meaningless){
+	//unsafe method call
+	meaningless.equals("wow");
```
Comment (cschroedl-gov) [inline src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java:42]: this method call is unsafe; asking for an NPE

Comment (cschroedl-gov) [inline src/test/java/gov/usgs/cida/ConsulRegistratorMojoTest.java:42]: :+1:

File: README.md (hunk 1 above)
Comment (cschroedl-gov) [inline README.md]: :+1:

**Comment by cschroedl-gov** [comment]
:shipit:

**Pull request closed without merging**